specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.0/FormRecognizer.json

```diff
@@ -506,6 +606,97 @@
       },
       "x-nullable": false
     },
+    "Authorization": {
+      "description": "Request parameter that contains claims for authorizing an operation.",
+      "required": [
+        "modelId"
+      ],
+      "type": "object",
+      "properties": {
+        "modelId": {
```

Comment:
>modelId [](start = 9, length = 7)

Does it mean I need to have model (any) on recipient side in order to copy something where?